pkg/controller/syncset/syncset_controller.go

```diff
@@ -23,7 +23,10 @@ import (
 
 	"github.com/openshift/hive/pkg/apis/helpers"
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+	"github.com/openshift/hive/pkg/constants"
 	controllerutils "github.com/openshift/hive/pkg/controller/utils"
+
+	k8slabels "k8s.io/kubernetes/pkg/util/labels"
```

Comment:
Put this import with the group of `k8s.io` imports.

Reply:
Fixed.